t9n/pt.coffee

```diff
@@ -42,16 +42,30 @@ pt =
   username: "Nome de usuário"
   usernameOrEmail: "Usuário ou e-mail"
   with: "com",
-  "Send the verification email again" : "Reenviar e-mail de verificação",
-  "Send email again" : "Reenviar e-mail"
```

Comment:
I did a diff with the english version and both wheren't there. I noticed the key value for resendVerificationEmail and I added it. Also both are the quite the same message.